catroid/src/main/java/org/catrobat/catroid/content/Sprite.java

```diff
@@ -664,4 +729,22 @@ public void updateUserVariableReferencesInUserVariableBricks(List<UserVariable>
 			}
 		}
 	}
+
+	public void setConvertToSingleSprite(boolean convertToSingleSprite) {
```

Comment:
Thx for the hint, but I'm not sure if it's easier - I use it mainly when elements are dragged and clone (or shallow clone) it there. There is e.g. the method swapAndCloneMobileSprite(..) in UtilDynamicListView where mainly SingleSprites and GroupItemSprites are converted. ConvertToSingleSprite and convertToGroupItemSprite are called before that function, so I don't know that yet inside the function.
Of course, I could pass an additional boolean function parameter, but I'm not sure if it gets so much easier with that.